cmd/root.go

```diff
@@ -187,55 +219,12 @@ func runCmd(args []string, g Getter, out io.Writer) error {
 	csvWriter.Flush()
 
 	if len(backoffQueue) > 0 {
-		sugar.Debugf("Reconciling back off queue: %s", backoffQueue)
+		zap.S().Debugf("Reconciling back off queue: %s", backoffQueue)
 	}
 
 	return nil
 }
 
-// rootCmd represents the base command when called without any subcommands
-var rootCmd = &cobra.Command{
-	Use:   "gh-dependency-report [flags] owner [repo ...]",
-	Short: "Generate report of repository manifests and dependencies discovered through the dependency graph",
-	Long:  "Generate report of repository manifests and dependencies discovered through the dependency graph",
-	Args:  cobra.MinimumNArgs(1),
-	RunE: func(cmd *cobra.Command, args []string) error {
-
-		var err error
-		var client api.GQLClient
-
-		client, err = gh.GQLClient(&api.ClientOptions{
-			Headers: map[string]string{
-				"Accept": "application/vnd.github.hawkgirl-preview+json",
-			},
-		})
-
-		if err != nil {
-			return err
-		}
-
-		return runCmd(args, newAPIGetter(client), os.Stdout)
-	},
-}
-
-// Execute adds all child commands to the root command and sets flags appropriately.
-// This is called by main.main(). It only needs to happen once to the rootCmd.
-func Execute() {
-	err := rootCmd.Execute()
-	if err != nil {
-		os.Exit(1)
-	}
-}
-
-func init() {
-	rootCmd.Flags().StringSliceVarP(&repoExcludes, "exclude", "e", []string{}, "Repositories to exclude from report")
-	rootCmd.Flags().StringVarP(&outputFile, "output-file", "o", "", "Name of file to write CSV report, defaults to stdout")
-
-	logger, _ := zap.NewDevelopment()
-	defer logger.Sync()
-	sugar = logger.Sugar()
-}
-
```

Comment:
These changes are a bit muddy given the others as I had difficulty of initializing the command's flags with the current structure much less help make the code more unit testable later.  The other changes in this PR were as minimal as I could make given where I was at the time.